sdk/core/azure-core/src/main/java/com/azure/core/implementation/annotation/Patch.java

```diff
@@ -17,14 +17,18 @@
  * <p><strong>Example 1: Relative path segments</strong></p>
  *
  * <pre>
- * {@literal @}Patch("subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}")
- *  VirtualMachine patch(@PathParam("resourceGroupName") String rgName, @PathParam("vmName") String vmName, @PathParam("subscriptionId") String subscriptionId, @BodyParam VirtualMachineUpdateParameters updateParameters);</pre>
+ * {@literal @}Patch("subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/
```

Comment:
In general, the method line breaks in these (annotations/*.java files) are weird for samples because I think these are supposed to represent how people would write a method in their code. 

I wouldn't split a parameter after the type.

Reply:
I think the `<pre>` blocks represent that they want it to be rendered as it is typed? So if that's the case, I'd try to make reasonable code splits.